static/docs/user-guide/install/mac.md

```diff
@@ -0,0 +1,20 @@
+# Installation on MacOS
+
+1. Install with `brew`:
+
+   ```dvc
+   $ brew install iterative/homebrew-dvc/dvc
+   ```
+
+   or:
+
+   ```dvc
+   $ brew cask install iterative/homebrew-dvc/dvc
+   ```
```

Comment:
Seems like we're deprecating this option per https://discordapp.com/channels/485586884165107732/565699007037571084/633702914103443458 (and https://github.com/iterative/homebrew-dvc/issues/16)